setup.sh

```diff
@@ -25,22 +26,43 @@ drupal8ci_install() {
 	# Add development dependencies to run the CircleCI jobs.
 	#
 	# behat/mink-extension is pinned until https://github.com/Behat/MinkExtension/pull/311 gets fixed.
-	composer require --dev \
-		cweagans/composer-patches \
-		behat/mink-extension:v2.2 \
-		behat/mink-selenium2-driver:^1.3 \
-		bex/behat-screenshot \
-		drupal/coder:^8.2 \
-		drupal/drupal-extension:master-dev \
-		drush/drush:~8.1 \
+	all_dev_deps=(
+		cweagans/composer-patches
```

Comment:
This one comes with drupal-composer, so we could remove it, right? It would simplify the setup.